src/app/accounting/search-journal-entry/search-journal-entry.component.ts

```diff
@@ -182,7 +194,7 @@ export class SearchJournalEntryComponent implements OnInit, AfterViewInit {
         debounceTime(500),
         distinctUntilChanged(),
         tap((filterValue) => {
-          this.applyFilter(this.dateUtils.getDate(filterValue), 'fromDate');
+          this.applyFilter(this.dateUtils.formatDate(filterValue, this.settingsService.dateFormat), 'fromDate');
```

Comment:
In general, `getDate` uses by default the `settingsService.dateFormat` value

Reply:
@josehernandezfintecheandomx It is not... 

  ```
public getDate(timestamp: any): string {
    return this.datePipe.transform(timestamp, 'YYYY-MM-DD');
  }
```